packages/core/src/components/Forms/WarningText/WarningText.tsx

```diff
@@ -67,7 +67,7 @@ export const HvWarningText = (props: HvWarningTextProps) => {
   const showWarning = localVisible && !localDisabled;
   const content = showWarning ? children : "";
   const localAdornment = adornment || (
-    <Fail className={classes.defaultIcon} color="negative" />
+    <Fail iconSize="XS" className={classes.defaultIcon} color="negative" />
```

Comment:
`XS` makes it SVG icon `12px`, which looks too small. It (looks like) is `16px` in the mocks.
From my understanding, only the container changes, not the SVG size 🤔 

```suggestion
    <Fail className={classes.defaultIcon} color="negative" />
```

Reply:
I will ask DS system about the size of the icon to be sure

Reply:
To clarify, to me it looks like a `16px` (`iconSize: "S"`) SVG icon on a `24px` container

Reply:
DS responded and said the icon size is `12px`. In this case, I think I should maintain `iconSize="XS"` and the override  for the `svg` dimensions in DS3 (comment below). Do you agree or suggest an alternative?

Reply:
If it's `12px`, then it's `12px` 👍  - I guess I'm bad at counting pixels